modules/shared/partials/acid-transactions.adoc

```diff
@@ -99,12 +103,11 @@ The cleanup process is detailed below in <<Asynchronous Cleanup>>.
 
 === Committing
 Only once the lambda has successfully run to conclusion, will the attempt be committed.
-This updates the attempt entry, which can be used as a signal by transactional actors as to whether to use the post-transaction version of a document from its XATTRs.
-Hence updating the ATR entry is effectively an 'atomic commit' switch for the transaction.
+This updates the ATR entry, which is used as a signal by transactional actors to use the post-transaction version of a document from its XATTRs.
```

Comment:
first use of the term ATR that I see.  Should it be defined here or should we make it more vague, like "this updates transaction metadata"?

Reply:
No I think this one's ok, it's defined a couple of sections up in 'Active Transaction Record Entries'